src/script/auth/page/Index.test.tsx

```diff
@@ -42,7 +43,16 @@ class IndexPage {
     store: MockStoreEnhanced<TypeUtil.RecursivePartial<RootState>, ThunkDispatch<RootState, Api, AnyAction>>,
     history?: History<any>,
   ) {
-    this.driver = mountComponent(<Index />, store, history);
+    this.driver = mountComponent(
+      <Index
+        title=""
+        changeTitle={title => {
+          document.title = title;
+        }}
```

Comment:
The idea would be more to pass on a `mock` function here and in the test check that the mock function has been called. 
In the end the component doesn't really care what side effect is happening on the consumer side. 

It allows you to test your component as a unit: you just test that the component is calling the function when it's rendered :)